GUI/MainModInfo.Designer.cs

```diff
@@ -20,6 +20,17 @@ protected override void Dispose(bool disposing)
             base.Dispose(disposing);
         }
 
+        /// <summary>
+        /// Create a new TransparentTextBox control that allows the backcolor to be transparent
+        /// </summary>
+        private partial class TransparentTextBox : System.Windows.Forms.TextBox
```

Comment:
Could you please put this class in a separate file? I think we should limit the .Designer.cs files to things that the VS editor code generation can handle.

Reply:
Ouups, forgot about bitchy VS :)